jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -265,6 +265,13 @@
     </rosparam>
   </node>
 
+  <!-- send email, twitter with smach -->
+  <node name="server_name" pkg="jsk_robot_startup" type="smach_to_mail.py" output="screen">
```

Comment:
smach topic name is normally `/server_name/smach/container_status`.
thats why `smach_to_mail` node's name is `server_name` in launch
but I think it is confusing and  a bit lazy solution...